publication-rest/src/test/java/no/unit/nva/publication/fetch/FetchPublicationHandlerTest.java

```diff
@@ -423,6 +426,17 @@ void shouldReturnRepublishAllowedOperationWhenReturningTombstoneAndUserHasAccess
         assertThat(actualPublication.getAllowedOperations(), hasItem(PublicationOperation.REPUBLISH));
     }
 
+    private Publication createUnpublishedPublication(WireMockRuntimeInfo wireMockRuntimeInfo) throws ApiGatewayException {
+        var publication = fromInstanceClassesExcluding(PermissionStrategy.PROTECTED_DEGREE_INSTANCE_TYPES);
```

Comment:
Can a Degree be unpublished at all? Maybe test if the person that can can do it?